archive/README.md

```diff
@@ -0,0 +1,58 @@
+# docker
```

Comment:
What's your idea behind keeping this as an archived file? For the normal use cases the git history is enough I think.

(Same for all files moved to "archive".)